airbyte-integrations/connectors/source-amazon-mws/source_amazon_mws/schemas/GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL.json

```diff
@@ -0,0 +1,158 @@
+{
+  "name": "GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL",
+  "supported_sync_modes": ["full_refresh", "incremental"],
+  "source_defined_cursor": true,
+  "default_cursor_field": ["purchase-date"],
+  "json_schema": {
+    "type": "object",
+    "title": "GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL",
+    "description": "Tab-delimited flat file report that shows all orders that were placed in the specified period. For all sellers. Can be requested.",
```

Comment:
since this is not necessarily loaded as a file, maybe the description can be different i.e: `order data`